app/annotationsList/annotationList.js

```diff
@@ -130,49 +149,27 @@ app.controller('AnnotationListCtrl', function ($rootScope, $scope, $http, $uibMo
     postProcessGoTerms(_.unique(goTermIds));
   }
 
-  function postProcessTaxa(taxaIds) {
-    $scope.taxaMapping = {};
-    if (taxaIds.length !== 0) {
-      var taxonomyPromise = taxonomyService.getTaxa(taxaIds);
-      taxonomyPromise.then(function (multipleTaxa) {
-        angular.forEach(multipleTaxa.data.taxonomies, function (taxon) {
-          $scope.taxaMapping[taxon.taxonomyId] = taxon;
-        });
-      });
-    }
-  }
-
-  function postProcessGeneProds(geneProductIds) {
-    $scope.gpMapping = {};
-    if (geneProductIds.length !== 0) {
-      var geneProdPromise = geneProductService.getGeneProducts(geneProductIds);
-      geneProdPromise.then(function (response) {
-        angular.forEach(response.data.results, function (geneProd) {
-          $scope.gpMapping[geneProd.id] = geneProd;
-        });
-      }, function (reason) {});
-    }
-  }
-
-  function postProcessGoTerms(goTermIds) {
-    $scope.goTermMapping = {};
-
-    termService.getGOTerms(goTermIds).then(function (resp) {
-      angular.forEach(resp.data.results, function (goTerm) {
-        $scope.goTermMapping[goTerm.id] = goTerm;
-      })
-    }, function (reason) {
-      console.log(reason);
-    });
-  }
-
   /**
    * Put commas between the rather large numbers we can have here.
    */
   function prettyPrintNumberAnnotations(numberAnnotations) {
     $scope.totalItems = numberAnnotations.toLocaleString();
   }
 
+  function getResultsPage() {
+    var query = $routeParams;
+    $scope.columns.slimmedTerm.visible = (query.goUsage && query.goUsage === 'slim');
+
+    $scope.resultsPromise = searchService.findAnnotations($scope.currentPage, $scope.maxSize,
+      searchService.serializeQuery(query));
+    $scope.resultsPromise.then(function (data) {
+      $scope.goList = data.data;
+      $scope.annotations = $scope.goList.results;
+
+      prettyPrintNumberAnnotations($scope.goList.numberOfHits);
```

Comment:
Why do we need a function here? Why not just 
$scope.goList.numberOfHits = $scope.goList.numberOfHits.toLocaleString();
?

And, if we do need a function, should not it be in a service so it can be used everywhere and then all numbers are formatted the same?

Another possibility is using filters on the html but then if we decide to change the format then it would be more code to be changed.